firstvoices/backend/models/characters.py

```diff
@@ -31,6 +32,12 @@ class Meta:
                 fields=["sort_order", "site_id"], name="unique_character_sort_order"
             ),
         ]
+        rules_permissions = {
+            "view": predicates.has_visible_site,
+            "add": predicates.is_superadmin,
+            "change": predicates.is_superadmin,
+            "delete": predicates.is_superadmin,
+        }
 
```

Comment:
Since you're adding permissions, could you please add them for the rest of the models in this file?

Reply:
Sure, since the alphabet model ill not be exposed in the API does that model require permissions as well?